.github/workflows/linting.yml

```
@@ -0,0 +1,80 @@
+---
+name: Linting
+
+on:
+  pull_request:
+    branches:
+      - '**'
+  push:
+    branches:
+      - master
+    tags:
+      - 'v**'
+
+jobs:
+  code-gen:
+    name: Submariner K8s API Code Generation
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@master
+      - name: Run validation
+        run: make codegen
+
+  dco:
+    runs-on: ubuntu-latest
+    name: DCO Check
+    steps:
+      - name: Get PR Commits
+        id: 'get-pr-commits'
+        uses: tim-actions/get-pr-commits@master
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: DCO Check
+        uses: tim-actions/dco@master
+        with:
+          commits: ${{ steps.get-pr-commits.outputs.commits }}
+
+  gitlint:
+    name: Gitlint
```

```suggestion
    name: Commit Message(s)
```
Perhaps it's clearer the way it was, or just 'Commit Messages' or something.

Describe they 'why' not the 'what' ;)

---

Good idea, done.